### DISCUSSION TITLE
Github actions (ci.yml) setup

### DISCUSSION THREAD

**Discussion by BirbEXE** [squidfunk/mkdocs-material#4034] [Q&A] [answered]
heya all.
I was trying to setup the github action for mkdocs-material, and I keep getting the error

```remote: Permission to git denied to github-actions[bot].```

[here's the deploy log](https://github.com/birb-studio/caffeine.docs/runs/6956635392?check_suite_focus=true)

I'm reasonably new to github actions, so I figure i've just missed something really easy ;-; 

could someone give me a hand?

**Comment by prashantdwivedi** [comment, marked as answer]
Hi [BirbEXE](https://github.com/BirbEXE)

I check your `cl.yaml` file on your file `python-version: 3.x`; just change with the latest version of python ` python-version: 3.8` and try again.

Today I face a similar issue bec I did not notice 
Please Note: I'm not a badass developer. I'm a learner by mistake 😉
